src/Components/AffectedClustersTable/AffectedClustersTable.spec.ct.js

```diff
@@ -183,7 +184,8 @@ describe('non-empty successful affected clusters table', () => {
   });
 
   it('renders table header', () => {
-    cy.get(TABLE).find('th').should('have.text', 'Name');
+    cy.get(TABLE).find('th').children().eq(0).should('have.text', 'Name');
```

Comment:
Please, merge the Iker's patch first #96 to your branch and consider using the helper functions from the newly created view object: https://github.com/RedHatInsights/ocp-advisor-frontend/pull/96/files#diff-cf954007ae7e45436bf54b4844c273ee0c88283a5d28460b39b7f3a5b64d86edR23. We now need to avoid writing such functions in each spec and start utilizing them from the single place (what Iker's done recently).

Reply:
Done